### PR TITLE
feat: remove developer device details settings selection view model [WPB-6441]

### PR DESCRIPTION
**Pull request by theswiftway**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-6441" title="WPB-6441" target="_blank">Sub-taskWPB-6441</a>  [iOS] Remove `DeveloperDeviceDetailsSettingsSelectionViewModel`
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Solutions

Removed debug settings for E2EI

#### How to Test
1. On a debug build.
2. Shake the device
3. Go to Developer settings
4. Look for Device Details Settings.
5.Expectation - It should not be found

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by caldrian — APPROVED** [review]

**Review by johnxnguyen — APPROVED** [review]

File: wire-ios/Wire-iOS/Sources/Developer/DeveloperTools/E2eIdentity/DeveloperDeviceDetailsSettingsSelectionView.swift [hunk 1]
```diff
@@ -1,60 +0,0 @@
...
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-// GNU General Public License for more details.
-//
-// You should have received a copy of the GNU General Public License
-// along with this program. If not, see http://www.gnu.org/licenses/.
-//
-
-import SwiftUI
-
-struct DeveloperDeviceDetailsSettingsSelectionView: View {
```
Comment (netbe) [inline wire-ios/Wire-iOS/Sources/Developer/DeveloperTools/E2eIdentity/DeveloperDeviceDetailsSettingsSelectionView.swift:21]: question: this was not used ?

File: wire-ios/Tests/Extensions/E2eIdentityCertificate+mocks.swift [hunk 2]
```diff
@@ -0,0 +1,71 @@
+//
```
Comment (netbe) [inline wire-ios/Tests/Extensions/E2eIdentityCertificate+mocks.swift]: nitpick: rename to E2eIdentityCertificate+Mocks.swift

**Review by netbe — APPROVED** [review]
just a question and nitpic, not blocking

File: wire-ios/Wire-iOS/Sources/Developer/DeveloperTools/E2eIdentity/DeveloperDeviceDetailsSettingsSelectionView.swift (hunk 1 above)
Comment (theswiftway) [reply wire-ios/Wire-iOS/Sources/Developer/DeveloperTools/E2eIdentity/DeveloperDeviceDetailsSettingsSelectionView.swift:21]: This is no longer needed as we would use the e2ei status from the server

**Pull request merged**